api/db/seeds/data/team-certification/shared/constants.js

```diff
@@ -0,0 +1,36 @@
+import {
+  CLEA_COMPLEMENTARY_CERTIFICATION_ID,
+  PIX_DROIT_COMPLEMENTARY_CERTIFICATION_ID,
+  PIX_EDU_1ER_DEGRE_COMPLEMENTARY_CERTIFICATION_ID,
+} from '../../common/complementary-certification-builder.js';
+
+const TEAM_CERTIFICATION_OFFSET_ID = 7000;
+
+// IDS
+/// USERS
+const TEAM_CERTIFICATION_OFFSET_ID_USERS = TEAM_CERTIFICATION_OFFSET_ID + 100;
+export const SHARED_ORGANIZATION_USER_ID = TEAM_CERTIFICATION_OFFSET_ID_USERS + 4;
+export const CERTIFIABLE_SUCCESS_USER_ID = TEAM_CERTIFICATION_OFFSET_ID_USERS + 5;
```

Comment:
```suggestion
export const SHARED_ORGANIZATION_USER_ID = TEAM_CERTIFICATION_OFFSET_ID_USERS + 1;
export const CERTIFIABLE_SUCCESS_USER_ID = TEAM_CERTIFICATION_OFFSET_ID_USERS + 2;
```

**question:** pourquoi cet incrément à partir de 4 ? (Même question pour les incréments des centres de certif)

Reply:
Parce que j'ai vire les increment entre deux (2,3..) xD du coup oui remise au propose possible !,

Reply:
Done